py/picca/pk1d/postproc_pk1d.py

```diff
@@ -349,6 +382,9 @@ def variance_function(snr, amp, zero_point):
                     mean = np.mean(p1d_table[col][select])
```

Comment:
shouldn't this line be in an else after line 386 in the new version?

Reply:
yes